cdap-common/src/main/java/co/cask/cdap/common/conf/Configuration.java

```diff
@@ -461,7 +464,7 @@ public void addResource(InputStream in) {
    * be read again before accessing the values. Values that are added
    * via set methods will overlay values read from the resources.
    */
```

Comment:
leave this method to be public? Might be intended to be available for future use outside this package.
Also, this class is modeled after Hadoop's Configuration class, and there, it is public.